go/vt/mysqlctl/s3backupstorage/s3.go

```diff
@@ -173,7 +173,7 @@ func (bh *S3BackupHandle) AddFile(ctx context.Context, filename string, filesize
 		})
 		object := objName(bh.dir, bh.name, filename)
 		sendStats := bh.bs.params.Stats.Scope(stats.Operation("AWS:Request:Send"))
-		_, err := uploader.UploadWithContext(ctx, &s3manager.UploadInput{
+		_, err := uploader.Upload(&s3manager.UploadInput{
```

Comment:
Should we leave a code comment here? Eventually someone else will probably stroll through this code and see an opportunity for improvement and end up breaking things again.

Reply:
@L3o-pold can you add the comment to avoid this change in future?